### ISSUE TITLE
set table modification timestamp

### ISSUE THREAD

**Issue by petri** [RedTurtle/collective.tablepage#30]
See http://docs.plone.org/develop/plone/content/timestamps.html

Basically, just have storage call ```setModificationDate(DateTime.DateTime())``` whenever a row is modified.

**Comment by petri** [comment]
This should probably be done in a subscriber to event sent by implementation for #20

**Comment by petri** [comment]
Ahh good point thx @keul - my immediate use case does not require searching by modification timestamp so forgot that :)

Looks like one can index just the modification date, too: `context.reindexObject(idxs=["modified"])`

**Comment by keul** [comment]
Yeah, but calling only `context.reindexObject(idxs=["something"])` will not update catalog metadata on old Plone (BTW: not sure if compatibility with old Plone still matter)

**Comment by petri** [comment]
@cekk could you please weigh in on this? do you know of customers using this on old Plone versions? @keul would you remember up to what versions of Plone suffer from the old behavior you described?

**Comment by keul** [comment]
@petri sorry I don't remember, but I'm quite sure that Plone 3 compatibility is not important nowadays

**Comment by cekk** [comment]
@petri i don't know, and as suggested by @keul i think we shouldn't care about Plone 3 compatibility.
